### PR TITLE
Handbook

### PR DESCRIPTION
**Pull request by mondus**
Adds the RSE Working Practice document to the website. Note: I have not built this locally.

### CODE REVIEW

File: pages/collaboration/handbook/handbook.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+
+
+* Regular (but not too regular) focused meetings to prioritise work to be carried out.
+* Agreeing on an instant messaging platform for quick discussions, e.g. Google Chat, Slack or Skype - this is particularly useful in our current remote/hybrid working model.
+* Use of issues and pull requests, described in more detail below.
+
+We encourage the use of instant messaging (for quick discussions) or slack for throwaway conversation. As a general rule we suggest that you should not expect or require anyone to read messages on an instant messaging service (e.g slack, google chat, etc) unless they are specifically tagged (either individually or via a channel). We also need decisions to always be recorded. Notes or decision making around particular development tasks should always be recorded in an agreed location (e.g. GitHub Issues, or a Google Drive folder) rather than via email history.
+
+
+### Issues & Project Boards {#issues-&-project-boards}
```
Comment (davidwyld) [inline pages/collaboration/handbook/handbook.md]: the `&` causes this to render the contents of the curly braces, change to `{#issues-and-project-boards}` or something without the ampersand

File: pages/collaboration/handbook/handbook.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+
+
+### Instructions for delivery of necessary materials to initiate project {#instructions-for-delivery-of-necessary-materials-to-initiate-project}
+
+Before work begins, it is likely that we will need to inspect existing materials. This may include code, example data, database schema etc.
+
+
+### Code {#code}
+
+The RSE team delivers code according to best practises using a collaborative version control system (preferably GitHub, but equivalent tools such as GitLab or Bitbucket can be used for established projects). At the outset of the project initial code can be provided on a secure shared resource like Google Drive however we would expect to migrate this to version control to make any changes. The RSE team can help you to do this. If you already use version control and have a private repository, you will need to add the allocated RSE as a collaborator.
```
Comment (davidwyld) [inline pages/collaboration/handbook/handbook.md]: practises -> practices

File: pages/collaboration/handbook/handbook.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
+---
+title: RSE Handbook (RSE Working Practice)
+permalink: /collaboration/handbook/
+slug: index
+type: text
+---
+
```
Comment (davidwyld) [inline pages/collaboration/handbook/handbook.md]: Would it be possible to have a table of contents? (can be left for a future PR though)

**Review by davidwyld — CHANGES_REQUESTED** [review]

**Comment by davidwyld** [comment]
There's a potential name clash within the team for the term 'handbook', though not confusing on the website itself. If you wanted to, could change the name of this to 'collaboration guide' or something.

File: pages/collaboration/handbook/handbook.md (hunk 3 above)
Comment (mondus) [reply pages/collaboration/handbook/handbook.md]: Probably but not sure how. Would rather megre and then do this as a new PR.

File: pages/collaboration/handbook/handbook.md (hunk 2 above)
Comment (mondus) [inline pages/collaboration/handbook/handbook.md]:
```suggestion
The RSE team delivers code according to best practices using a collaborative version control system (preferably GitHub, but equivalent tools such as GitLab or Bitbucket can be used for established projects). At the outset of the project initial code can be provided on a secure shared resource like Google Drive however we would expect to migrate this to version control to make any changes. The RSE team can help you to do this. If you already use version control and have a private repository, you will need to add the allocated RSE as a collaborator.
```

File: pages/collaboration/handbook/handbook.md (hunk 1 above)
Comment (mondus) [inline pages/collaboration/handbook/handbook.md]:
```suggestion
### Issues and Project Boards {#issues-and-project-boards}
```

Comment (mondus) [reply pages/collaboration/handbook/handbook.md]: Changes made

File: pages/collaboration/handbook/handbook.md (hunk 2 above)
Comment (mondus) [reply pages/collaboration/handbook/handbook.md]: fixed

**Comment by mondus** [comment]
Changes made and renamed as collaboration guide. @davidwilby Can you resolve if happy and give it a final check.

**Comment by davidwyld** [comment]
> Changes made and renamed as collaboration guide. @davidwilby Can you resolve if happy and give it a final check.

Ha! Sorry, I was just this moment going to the website to see if this had been merged. Have resolved. LGTM

**Review by davidwyld — APPROVED** [review]

**Pull request merged**